packages/mediapipe-task-text/analysis_options.yaml

```diff
@@ -0,0 +1,8 @@
+include: ../analysis_options.yaml
+
+# Additional information about this file can be found at
+# https://dart.dev/guides/language/analysis-options
+
```

Comment:
Should this include the public member docs lint like in the core package?

Reply:
It sure should.